client/polyfills.js

```diff
@@ -0,0 +1,5 @@
+import '@babel/polyfill';
```

Comment:
Please migrate away from `@babel/polyfill` to `core-js` replacement :bowtie:

Reply:
That's addressed in #352 . It's out scope of this PR anyway.

Reply:
Hmm let's see:
1. This PR bumps `babel` to at least 7.6.0
2. Quoting babel polyfill [docs](https://babeljs.io/docs/en/babel-polyfill) here:
>As of Babel 7.4.0, this package has been deprecated in favor of directly including core-js/stable (to polyfill ECMAScript features) and regenerator-runtime/runtime (needed to use transpiled generator functions)

So although it has been part of vue-cli migration it should actually happen right away :wink: